website/docs/docs/collaborate/explore-projects.md

```diff
@@ -51,75 +47,107 @@ By default, dbt Explorer shows the project's [applied state](/docs/dbt-cloud-api
-- `tag:` &mdash; Find resources by a specified [tag](/reference/node-selection/methods#the-tag-method).
+<expandable alt_header="List of available lenses">
 
-<VersionBlock firstVersion="1.5">
+- **Default** (resource type)
+- **Materialization Type** (for example, identify incremental model dependencies)
+- **Lastest Status** (for example, diagnose a failed DAG region)
+- **Model Layer** (for example, discover marts models to analyze)
+    - **Marts** &mdash; A model with the prefix `fct_` or `dim_`. Or, a model that lives in the `/marts/` subdirectory.
+    - **Intermediate** &mdash; A model with the prefix `int_`. Or, a model that lives in the `/int/` or `/intermediate/` subdirectory.
+    - **Staging** &mdash; A model with the prefix `stg_`. Or, a model that lives in the `/staging/` subdirectory.
 
-- `group:` &mdash; Find models defined within a specified [group](/reference/node-selection/methods#the-group-method).
-- `access:` &mdash; Find models based on their [access](/reference/node-selection/methods#the-access-method) property.
+</expandable>
 
-</VersionBlock>
+### Example of lenses
 
-When searching with selector methods, you can also use [graph operators](/reference/node-selection/graph-operators). For example, `+orders` returns all the upstream nodes of `orders` and `orders+1` returns just the nodes immediately downstream (the children) of `orders`.
+Example of applying the **Materialization Type** _lens_ with the lineage graph significantly zoomed out: 
 
-You can use multiple selector methods in your search query with [set operators](/reference/node-selection/set-operators). A space implies a union set operator and a comma for an intersection. For example:
-- `resource_type:metric,tag:nightly` &mdash; Returns metrics with the tag `nightly`
-- `+snowplow_sessions +fct_orders` &mdash; Returns resources that are parent nodes of either `snowplow_sessions` or `fct_orders`
+<Lightbox src="/img/docs/collaborate/dbt-explorer/example-materialization-type-lense.png" width="100%" title="Example of the Materialization type lens" />
 
-<LoomVideo id='712c8b74a6e242699ad2d32e2b0693a5' />
 
-## Browse with the sidebar
+## Keyword search {#search-resources}
+
+You can locate resources in your project by performing keyword search in the search bar. All resource names, column names, resource descriptions, warehouse relations, and code that match your search criteria will display as a list on the main (center) section of the page. When searching for an exact column name, the results show all relational nodes containing that column in their schemas. If there's a match, a notice in the search result indicates the resource contains the specified column. Also, you can apply filters to further refine your search results.
+
+<expandable alt_header="Search features">
+
+- **Partial keyword search** &mdash; This is also referred to as [fuzzy search](https://en.wikipedia.org/wiki/Approximate_string_matching).
```

Comment:
Does it matter that neither of the terms used in this sentence are the title of the page we link to? Should we add a bit
```suggestion
- **Partial keyword search** &mdash; This is also referred to as [approximate string matching, or a fuzzy search](https://en.wikipedia.org/wiki/Approximate_string_matching).
```